Tests/Test Helpers/Editor/EditorVRTestInitializer.cs

```diff
@@ -0,0 +1,35 @@
+﻿using NUnit.Framework;
+using UnityEditor.Experimental.EditorVR.Core;
+
+namespace UnityEditor.Experimental.EditorVR.Tests.Core
+{
+	[SetUpFixture]
+	public class EditorVRTestInitializer
+	{
+		EditingContextManagerSettings projectSettingsBackup;
+		EditingContextManagerSettings userSettingsBackup;
+
+		[OneTimeSetUp]
+		public void SetupBeforeAllTests()
+		{
+			projectSettingsBackup = EditingContextManager.LoadProjectSettings();
+			userSettingsBackup = EditingContextManager.LoadUserSettings();
+
+			EditingContextManager.ShowEditorVR();
```

Comment:
I suppose this means any of these tests will cover whether EVR can start without errors?